src/main/java/edu/kis/powp/jobs2d/drivers/gui/DriverChangeObserver.java

```diff
@@ -0,0 +1,12 @@
+package edu.kis.powp.jobs2d.drivers.gui;
+
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.observer.Subscriber;
+
+public class DriverChangeObserver implements Subscriber {
```

Comment:
All DriverManager observers will be driver change observers. The name of observer should also describe its activity. Please rename.